generic/MaplGeneric.F90

```diff
@@ -2,7 +2,6 @@ module MaplGeneric
    use mapl_AbstractComponent
    use mapl_MaplComponent
    use mapl_ComponentSpecification
-   use mapl_VariableSpecification
```

Comment:
I still don't think this change is required.   And I'll likely need to put it right back as soon as I continue refactoring MAPL.

Reply:
UFS-GOCART builds correctly after reintroducing this use statement  Thanks, @tclune, for catching this unnecessary change. Apologies for the mix-up.